mindlessgen.toml

```diff
@@ -42,6 +42,8 @@ element_composition = "C:2-3, H:1-2, O:1-2, N:1-*"
 # > CAUTION: This option is overridden by the 'element_composition' option.
 # > I.e., if an element is specified in 'element_composition' with an occurrence > 0, it will be added to the molecule anyway.
 forbidden_elements = "57-71, 81-*"
+# > Set a charge for the molecule generation. Options: ""(turn it off), "int"
+molecular_charge = ""
```

Comment:
Can't we set it to `none`? Just like you can set booleans to `true` or `false`.

Reply:
If not, we have to re-define the `setter` of this specific configuration.

Reply:
It is possible to use None in the String. But not None like a boolean.

Reply:
Then just re-define the corresponding `setter`. (Example: if the `string.lower()` equals to `none.lower()`, set it to `None` – Do you know what I mean?)

Reply:
`    @molecular_charge.setter
    def molecular_charge(self, molecular_charge_str: str):
        """
        Set the molecular_charge.
        """
        if molecular_charge_str.lower() == "none":
            return None
        else:
            molecular_charge = int(molecular_charge_str)
        if not isinstance(molecular_charge, int | None):
            raise TypeError("Fixed charge should be an integer.")
        self._molecular_charge = molecular_charge`

something like this?

Reply:
> ` if molecular_charge_str.lower() == "none":` something like this?

Yeah, like:
```python
if molecular_charge_str.lower() == "none":
   self.molecular_charge = None
```